java-monitoring/run_protoc.sh

```diff
@@ -0,0 +1,709 @@
+"-Igoogle/monitoring/v3/service.proto=google/monitoring/v3/service.proto" \
+"-Igoogle/monitoring/v3/service_service.proto=google/monitoring/v3/service_service.proto" \
+"-Igoogle/monitoring/v3/snooze.proto=google/monitoring/v3/snooze.proto" \
+"-Igoogle/monitoring/v3/snooze_service.proto=google/monitoring/v3/snooze_service.proto" \
+"-Igoogle/monitoring/v3/span_context.proto=google/monitoring/v3/span_context.proto" \
+"-Igoogle/monitoring/v3/uptime.proto=google/monitoring/v3/uptime.proto" \
+"-Igoogle/monitoring/v3/uptime_service.proto=google/monitoring/v3/uptime_service.proto" \
+"-Igoogle/api/annotations.proto=google/api/annotations.proto" \
+"-Igoogle/api/http.proto=google/api/http.proto" \
+"-Igoogle/protobuf/descriptor.proto=google/protobuf/descriptor.proto" \
+"-Igoogle/api/client.proto=google/api/client.proto" \
+"-Igoogle/api/launch_stage.proto=google/api/launch_stage.proto" \
+"-Igoogle/protobuf/duration.proto=google/protobuf/duration.proto" \
+"-Igoogle/api/distribution.proto=google/api/distribution.proto" \
+"-Igoogle/protobuf/any.proto=google/protobuf/any.proto" \
+"-Igoogle/protobuf/timestamp.proto=google/protobuf/timestamp.proto" \
+"-Igoogle/api/field_behavior.proto=google/api/field_behavior.proto" \
+"-Igoogle/api/label.proto=google/api/label.proto" \
+"-Igoogle/api/metric.proto=google/api/metric.proto" \
+"-Igoogle/api/monitored_resource.proto=google/api/monitored_resource.proto" \
+"-Igoogle/protobuf/struct.proto=google/protobuf/struct.proto" \
+"-Igoogle/api/resource.proto=google/api/resource.proto" \
+"-Igoogle/rpc/status.proto=google/rpc/status.proto" \
+"-Igoogle/type/calendar_period.proto=google/type/calendar_period.proto" \
+"-Igoogle/protobuf/empty.proto=google/protobuf/empty.proto" \
+"-Igoogle/protobuf/field_mask.proto=google/protobuf/field_mask.proto" \
+"-Igoogle/protobuf/wrappers.proto=google/protobuf/wrappers.proto" \
+google/monitoring/v3/alert.proto \
+google/monitoring/v3/alert_service.proto \
+google/monitoring/v3/common.proto \
+google/monitoring/v3/dropped_labels.proto \
+google/monitoring/v3/group.proto \
+google/monitoring/v3/group_service.proto \
+google/monitoring/v3/metric.proto \
+google/monitoring/v3/metric_service.proto \
+google/monitoring/v3/mutation_record.proto \
+google/monitoring/v3/notification.proto \
+google/monitoring/v3/notification_service.proto \
+google/monitoring/v3/query_service.proto \
+google/monitoring/v3/service.proto \
+google/monitoring/v3/service_service.proto \
+google/monitoring/v3/snooze.proto \
+google/monitoring/v3/snooze_service.proto \
+google/monitoring/v3/span_context.proto \
+google/monitoring/v3/uptime.proto \
+google/monitoring/v3/uptime_service.proto
+
+##################### Section 2 #####################
+# generate grpc-google-cloud-monitoring-v3-java-srcs_pkg.tar.gz
+#####################################################
+"${PROTOC_ROOT}"/protoc "--plugin=protoc-gen-rpc-plugin=${LIBRARY_GEN_OUT}/protoc-gen-grpc-java" \
```

Comment:
This is the command that does not generate the file with comments.

Reply:
The input File Descriptor content is different.

## With run_protoc.sh (that does not generate the comments)

```
suztomo@suztomo:~/google-cloud-java/java-monitoring$ grep 'The AlertPolicyService API is used to manage' "${LIBRARY_GEN_OUT}"/google/monitoring/v3/monitoring_proto-descriptor-set.proto.bin
suztomo@suztomo:~/google-cloud-java/java-monitoring$ 
```

This monitoring_proto-descriptor-set.proto.bin is 50 kB.

## With Bazel's sandbox directory (that generates the comments)

```
suztomo@suztomo:~/.cache/bazel/_bazel_suztomo/c9601419e42dcd7b8b40ef90684cc9a1/execroot/com_google_googleapis$ grep 'The AlertPolicyService API is used to manage' bazel-out/k8-fastbuild/bin/google/monitoring/v3/monitoring_proto-descriptor-set.proto.bin
grep: bazel-out/k8-fastbuild/bin/google/monitoring/v3/monitoring_proto-descriptor-set.proto.bin: binary file matches
```

This monitoring_proto-descriptor-set.proto.bin is 243 kB.

Reply:
Adding `--include_source_info` when generating *descriptor-set.proto.bin resolve this issue.

Reply:
Nice.